starboard/evergreen/shared/launcher.py

```diff
@@ -277,26 +277,24 @@ def _StageTargetsAndContentsRaspi(self):
 
   def _StageTargetsAndContentsRdk(self):
     """Stage targets and their contents for GN builds for RDK platforms."""
-    content_subdir = os.path.join('usr', 'share', 'cobalt')
 
     # The rdk loader always loads libcobalt.so. Hard code target_name to match.
     target_name = 'cobalt'
 
     # Copy target content and binary.
-    target_install_path = os.path.join(self.out_directory, 'install')
     target_staging_dir = os.path.join(self.staging_directory, 'content', 'app',
                                       target_name)
     os.makedirs(target_staging_dir)
 
     # TODO(b/218889313): Reset the content path for the evergreen artifacts.
-    content_subdir = os.path.join('usr', 'share', 'cobalt')
-    target_content_src = os.path.join(target_install_path, content_subdir)
+    target_content_src = os.path.join(self.out_directory, 'content')
     target_content_dst = os.path.join(target_staging_dir, 'content')
     shutil.copytree(target_content_src, target_content_dst)
     shlib_name = f'lib{self.target_name}.so'
```

Comment:
Can you also move this row down one? I think it belongs with the rowes below more than the one above.

Reply:
done.